### ISSUE TITLE
Labeling cutoff issues

### ISSUE THREAD

**Issue by schwanksta** [datadesk/osm-quiet-la#27]
![labeling](https://f.cloud.github.com/assets/20067/1409461/24a45676-3d93-11e3-8635-a523e10257f6.png)

**Comment by palewire** [comment]
This is the worst thing about our current maps and the highest priority bug, IMHO.

**Comment by schwanksta** [comment]
Theoretically, adjusting the metatile size is supposed to fix this -- https://www.mapbox.com/tilemill/docs/guides/metatiles/

But I never had any luck changing the size. Perhaps tilemill is better with metatiles now.

**Comment by ldegroot** [comment]
Agreed. I spent a considerable amount of time the last couple of days trying to figure this out. My take:

·         Text-min-padding (or lack of it) seems to be the culprit. I have been able to reduce the problem to about 5-10% simply by adjusting the padding on every level of street label.

·         The down-side is that to get them to display regularly, the street labels around them have to be removed. This is the function of text-min-padding.

·         If we aren’t willing to sacrifice street labels at zoom 15-16, we should probably not display the neighborhood levels there.

·         This is actually A problem at zoom 13-14 too. I can get most of them to display by dropping secondary road labels, which I don’t like.

·         Finally, the only way I found to see ALL the neighborhood labels to set text-allow-overlap to true. Looks OK in some cases and really bad in others.

The radical solution: Drop the neighborhood labels.

·         It seems like most cases where we reference neighborhoods directly, we provide a polygon.

·         If they don’t appear in every zoom level, their value diminishes.

·         What do we really lose by dropping them?

Since I’m new to this project, I’m sure there are facets that I’m unaware of. Consider this a starting point.

After that, inconsistent highway shields drive me crazy.

Len
